### PR TITLE
Adapt to new workflow chunk api

### PR DESCRIPTION
**Pull request by svanoort**
Adapts to latest changes to the chunk APIs, adds some helper methods, and makes the statuses use null and not unknown. 

Downstream of some of the changes in https://github.com/jenkinsci/workflow-api-plugin/pull/6, specifically https://github.com/jenkinsci/workflow-api-plugin/pull/6/commits/9efad20670d2989d519785e4711bdf7e627e7c6d

Upstream of an as-yet-un-pushed stageview PR.

@reviewbybees

### CODE REVIEW

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

File: pom.xml [hunk 1]
```diff
@@ -145,7 +145,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-release-plugin</artifactId>
                 <configuration>
-                    <tagNameFormat>pipeline-stage-view-@{project.version}</tagNameFormat>
+                    <tagNameFormat>pipeline-graph-analysis-@{project.version}</tagNameFormat>
```
Comment (jglick) [inline pom.xml]: :ant: Delete this whole section. You probably copy-pasted it from `pipeline-stage-view-plugin`, which was a multimodule repository.

File: pom.xml [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
+    <name>Pipeline Graph Analysis Plugin</name>
     <!--<url>https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Graph+Analysis+Plugin</url>-->
 
     <scm>
-        <connection>scm:git:git://github.com/svanoort/pipeline-graph-analysis-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:svanoort/pipeline-graph-analysis-plugin.git</developerConnection>
-        <url>https://github.com/svanoort/pipeline-graph-analysis-plugin/</url>
+        <connection>scm:git:git://github.com/jenkinsci/pipeline-graph-analysis-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/pipeline-graph-analysis-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/pipeline-graph-analysis-plugin/</url>
```
Comment (jglick) [inline pom.xml]: :ant: conventional is to use `${project.artifactId}-plugin`

**Comment by jglick** [comment]
No clue what this means but :bee:

**Comment by jglick** [comment]
(presumably this is downstream of something not linked to here)

File: pom.xml (hunk 1 above)
Comment (svanoort) [inline pom.xml]: Probably, should be handle automatically I imagine

File: pom.xml (hunk 2 above)
Comment (svanoort) [inline pom.xml]: done

**Comment by jglick** [comment]
re:bee:

**Pull request merged**